testbed/tests/test_images.py

```diff
@@ -130,3 +130,31 @@ async def test_save(app):
                 # suite is run; and the files have a PID in them to ensure that
                 # they're unique across test runs.
                 pass
+
+
+async def test_native_image(app):
+    # Generate an image using pillow
+    pil_image = PIL_Image.new("RGBA", size=(110, 30))
+    draw_context = PIL_ImageDraw.Draw(pil_image)
+    draw_context.text((20, 10), "Hello World", fill="green")
+
+    buffer = io.BytesIO()
+    pil_image.save(buffer, format="png", compress_level=0)
+
+    # Construct a Toga image from PIL image
+    image1 = toga.Image(buffer.getvalue())
```

Comment:
This step shouldn't be needed - Toga can accept PIL images directly as input (and this would be a good in-practice test of that).

Reply:
Also - more meaningful variable names wouldn't hurt; `orig_image` instead of `image1` in this case.

Reply:
Done. Thanks.